### ISSUE TITLE
Non-working search query

### ISSUE THREAD

**Issue by lrayshel** [cca/millennium_catalog#33]
The search query for "College Publications" (http://library.cca.edu/search/dcca%20ccac%20csac%20publications) on http://libraries.cca.edu/collections results in 'no matches found'. 

However, a better and functional search appears on http://libraries.cca.edu/special-collections: http://library.cca.edu/search~S7/dcca+c+p

Recommendation: change search on http://libraries.cca.edu/collections to http://libraries.cca.edu/special-collections: http://library.cca.edu/search~S7/dcca+c+p.

**Comment by phette23** [comment]
Hmm, while it does show "no matches found" that first search actually shows more adjacent subject headings than the latter. I see all these which are roughly CCA publications:

```
	Cca C Publications	 	2
 	Cca C Staff Works By Or About -- See Kauppi, Sherry.	 	1
	Cca C Student Publications	 	3
 	Cca C Visiting Faculty Works By Or About -- 3 Related Subjects	 	3
Your entry cca ccac csac publications would be here -- Search as Words
	Cca Publications	 	8
	Cca Publications Exhibitions  	c2009	1
 	Ccac Cca Faculty -- 2 Related Subjects	 	2
	Ccac Publications	 	13
```

while the second search only shows 2:

```
	Cca C Periodicals	 	36
	Cca C Publications	 	2
```

I'm going to change the link but I also wonder if those last two should be collapsed so we can just go straight to results without an intermediate step. I'll email a link to this issue to Teri and Amber to see if they have ideas about how to improve this.

**Comment by lrayshel** [comment]
Agreed that it would be ideal if users could be taken straight to a set of
search results.

**Comment by phette23** [comment]
FYI there's a huge list of local call numbers, some of which are closely related and need to be combined. This is likely going to be a summer project involving the cataloging staff.
